tests/integration/regions/navbar.region.js

```diff
@@ -0,0 +1,11 @@
+/* global $ */
+
+"use strict";
+
+class NavBar {
+	/* Represents the navbar */
+
+	get signIn() { return $("#sign-in-btn"); }
```

Comment:
Just noticed this file uses tabs. Not sure if ESLint was catching that, or if we maybe need to tweak the ESLint config (which is well outside the scope of this PR, but I'm just thinking out loud).

Reply:
Wrong size tabs?

Reply:
Wrong size indeed. Darn Visual Studio